tvb_build/build_step1.py

```diff
@@ -53,7 +53,7 @@
 FW_FOLDER = os.path.join(TVB_ROOT, 'tvb_framework')
 LICENSE_PATH = os.path.join(FW_FOLDER, 'LICENSE')
 RELEASE_NOTES_PATH = os.path.join(TVB_ROOT, 'tvb_documentation', 'RELEASE_NOTES')
-DATA_SRC_FOLDER = os.path.dirname(tvb_data.__file__)
+DATA_SRC_FOLDER = TVBZenodoDataset().extract_dir
```

Comment:
This would be an empty folder, right ?
Then I think we should rewrite the functions bellow using this `_copy_dataset` and `copy_distribution_dataset`. or rather remove them  .... ?